cmd/containerd-shim-runhcs-v1/delete.go

```diff
@@ -44,9 +66,16 @@ The delete command will be executed in the container's bundle as its cwd.
 		// log those messages (if any) on stderr so that it shows up in containerd's log.
 		// This should be done as the first thing so that we don't miss any panic logs even if
 		// something goes wrong during delete op.
-		logs, err := ioutil.ReadFile(filepath.Join(bundleFlag, "panic.log"))
-		if err == nil && len(logs) > 0 {
-			logrus.WithField("log", string(logs)).Error("found shim panic logs during delete")
+		// The file can be very large so read only first 1MB of data.
```

Comment:
For my own understanding, have we actually seen the file be very large? Anytime I'd see it pop up it was generally fairly small with only the stack of a single goroutine inside

Reply:
No usually this file is never that large. The only case that I can think of is if something is logging to stderr.

Reply:
That's what I was trying to figure out, do we ever actually log anything to stderr?

Reply:
We don't but there might be some code from imported packages that can write to stderr and that's not always in our control.

Reply:
Ahhh this slipped my mind, thank you for this that's a good point